### ISSUE TITLE
[Question] Is it possible to replace fastCloneDeep JSON.stringify(JSON.parse with structuredClone() ?

### ISSUE THREAD

**Issue by kievsash** [formio/angular#1016]
Hi
REASON to ask:
I am implementing custom angular file-select component and want File object to be accessible as formfield value on formSubmit. But when I do valueChanges.emit({file: someFileObjWithBlob}) inside custom element then on submit event I get empty object (btw customEvent transfer File obj correctly)

After small ivestigation I suspect that maybe it is because in [submit event handler](https://github.com/formio/angular/blob/master/projects/angular-formio/src/FormioBaseComponent.ts#L389) we sometimes do copy with [fastCloneDeep Util function](https://github.com/formio/formio.js/blob/master/src/utils/utils.js#L1339C17-L1339C30) which is just JSON.parse(JSON.stringify(obj))
So maybe it make sense to replace it with [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone)?
structuredClone does File structure copy correctly.

What do you think?
<img width="844" alt="image" src="https://github.com/formio/angular/assets/15340874/1bbd535c-c74e-4aea-a57c-92c958a85f68">
<img width="889" alt="image" src="https://github.com/formio/angular/assets/15340874/4d3f8a77-7e66-4bc5-8406-50b5963f8cda">

**Comment by lane-formio** [comment]
<div>Agent Viktoryia Alennikava linked Freshdesk ticket <a href="https://formhelp.freshdesk.com/a/tickets/52262">52262</a> for this issue.</div><br/>

**Comment by travist** [comment]
Yes, please submit a pull request with this change and we would be happy to merge it. Another alternative is that we are releasing a new "slim" angular module called "@formio/angular/embed". This may be another good option since we will be officially releasing that soon.

**Comment by kievsash** [comment]
Hi @travist 
Thank you for answering 
here is a pr: https://github.com/formio/formio.js/pull/5942

**Comment by johnformio** [comment]
Had to revert pr to use structured clone.  Reopening this to find another solution.
